### DISCUSSION TITLE
Tools / Options / Font & Colors - change color problems

### DISCUSSION THREAD

**Discussion by DKnoto** [albilu/netbeansPython#34] [General]
Today I wanted to adjust the syntax coloring to what I have in my jEdit editor. Unfortunately, my attempt failed. During this operation I discovered two problems in the Python handling module:

- selecting a section, e.g. comments, does not signal that the operation was successful. For Java in such a situation, the selected section flashes underlined text in the selected section;
- changing the color has no effect on the sample in the dialog box or in the actual Python code document.

Below are screenshots depicting this situation:

[Java before change](https://www.dropbox.com/s/z5apklljpj7jj6e/Font-and-Colors-Problem-01-Java-Before-Change.png?dl=0)
[Java after change](https://www.dropbox.com/s/r04cr2yb4t2pg9u/Font-and-Colors-Problem-02-Java-After-Change.png?dl=0)
[Python before change](https://www.dropbox.com/s/60c16570nwo21rv/Font-and-Colors-Problem-03-Python-Before-Change.png?dl=0)
[Python after change](https://www.dropbox.com/s/thwf34s5v9uibpd/Font-and-Colors-Problem-04-Python-After-Change.png?dl=0)

**Comment by albilu** [comment]
Indeed. This is a known issue already described here: #10 . Seems to be an issue with Textmate Languages in Netbeans. Need to discuss with the devs to see how to solve it.

In fact if you want to modify it you need to use this one instead 
![image](https://user-images.githubusercontent.com/35330562/236660593-8f1c60b1-0f99-4f61-9c76-c90067ec7d42.png)

the drawback is that the modification will apply also for all other Textmate based languages you have setted in your Netbeans

**Reply by DKnoto** [reply]
Thank you for the clarification. It seems that the NetBeans team is overloaded and does not give proper attention to supporting languages other than Java/PHP/JS. Recently, I was fighting with "new simple test" test handling errors for C/C+ in projects supported by the old module from NB 8.2. I reported the error but was not heard, it was stated that it was a bug in the old module and they don't have time to fix it. After analyzing your module for Python, I came to the conclusion that the error lies elsewhere. It turned out that the matter is trivial, the error lies in the new NB. The old NB before version 12.4 analyzed stdout output and the new one analyzes stderr for %%SUITE_*%%. They could at least correct this in the old documentation....
